service/src/main/java/bio/terra/workspace/service/workspace/model/Workspace.java

```diff
@@ -118,6 +118,23 @@ public int hashCode() {
         .toHashCode();
   }
 
+  /**
```

Comment:
This won't be needed if we build the destinationWorkspace object in the controller.